MapboxCoreNavigation/Feedback.swift

```diff
@@ -92,20 +92,14 @@ public enum FeedbackType: CustomStringConvertible {
             return "cars_not_allowed_on_street"
         case .illegalRoute(subtype: .turnAtIntersectionUnprotected):
             return "turn_at_intersection_was_unprotected"
-        case .illegalRoute(subtype: .other):
-            return "illegal_route"
-        case .roadClosure(subtype: .none):
-            return "road_closure"
         case .roadClosure(subtype: .streetPermanentlyBlockedOff):
             return "street_permanently_blocked_off"
         case .roadClosure(subtype: .roadMissingFromMap):
             return "road_is_missing_from_map"
-        case .roadClosure(subtype: .other):
-            return "road_closure"
-        case .positioning(subtype: .none):
-            return "positioning_issue"
         case .positioning(subtype: .userPosition):
             return "positioning_issue"
+        default:
+            return nil
```

Comment:
Ideally we could avoid `default` cases so that someone adding another category in the future can rely on the compiler’s switch exhaustiveness check to find this switch statement. But if explicitly mapping all the `other` and `none` cases to `nil` would be inconvenient, I guess we can leave it this way.